pkg/cli/mirror/fbc_operators.go

```diff
@@ -38,7 +41,6 @@ const (
 	configPath          string = "configs/"
 	catalogJSON         string = "/catalog.json"
 	relatedImages       string = "relatedImages"
-	configsLabel        string = "operators.operatorframework.io.index.configs.v1"
```

Comment:
It is safer to use the configsLabel: the configs folder is not always called this way as we learned a few months back

Reply:
Never mind this comment